scanpullrequest/scanpullrequest_test.go

```diff
@@ -1262,3 +1168,163 @@ func redirectLogOutputToNil() (previousLog log.Log) {
 	log.SetLogger(newLog)
 	return previousLog
 }
+
+type TestResult struct {
+	Sca     int
+	Iac     int
+	Secrets int
+	Sast    int
+}
+
+func TestAuditDiffInPullRequest(t *testing.T) {
+	tests := []struct {
+		testName       string
+		projectName    string
+		configPath     string
+		expectedIssues TestResult
+	}{
+		{
+			testName:    "Project with Jas issues (issues added removed and not changed)",
```

Comment:
I didn't understand the test name. Does the project we scan include all of these cases?

Reply:
yes